docs/tutorials/motor_noise_debugging.rst

```diff
@@ -0,0 +1,196 @@
+locations and the three phase motor cabling will extend along the arm to reach the motor itself. A simplified diagram of a drone wired with external ESCs can be seen below 
+in figure 2.
+
+.. figure:: ../_static/tutorial_images/motor_noise/external_esc_comms.png
+    :align: center
+
+    Figure 2: Simplified Drone Cabling Diagram With External ESCs
+
+Conversely, a drone utilizing motors with internal ESCs will have communications cables extending the entire length of the drone arm to reach the motor as seen below in 
+figure 3. Generally speaking, a communication network’s susceptibility to EMI will increase the longer its wire lengths get. Therefore, it is especially important to reduce 
+EMI and build robust communication buses on a vehicle utilizing motors with internal ESCs. 
+
+.. figure:: ../_static/tutorial_images/motor_noise/internal_esc_comms.png
+    :align: center
+
+    Figure 3: Simplified Drone Cabling Diagram With Internal ESCs
+
+Identifying Noise On Communication Lines
+=====================================================
+When an electric motor is mounted to a conductive frame with metal or conductive screws, the motor body and vehicle frame will be electrically connected. In this scenario, 
+it is likely that any motor noise that couples to the motor’s aluminum body via parasitic capacitances will conduct onto the vehicle’s frame. Communication buses that run 
+through or parallel to a noisy vehicle frame will be susceptible to the interference and may experience dropped packets or communication failures. The oscilloscope screen 
+capture below in figure 4 displays how motor noise that has been conducted onto a carbon fiber tube can interfere with signal buses that are routed through it. A simulated 
+drone setup was used to capture the scope images in this document can be seen below in figure 5. In this example the carbon fiber tubes used to simulate drone arms are one 
+meter in length and electrically connected. 
+
+.. figure:: ../_static/tutorial_images/motor_noise/noise_diagram.png
+    :align: center
+
+    Figure 4: Coupled Motor Noise 
+
+.. figure:: ../_static/tutorial_images/motor_noise/simulated_drone_setup.jpg
+    :align: center
+
+    Figure 5: Simulated Drone Communication Noise Setup 
+
+Channels 2 and 3 (light blue and pink) in figure 4 display CAN bus high and low signals at a recessive voltage state (i.e. no signal is actively being transmitted). 
+Channel 4 (dark blue) is measuring motor noise that has been conducted on to the carbon fiber tube “drone arms”. One can clearly observe the motor noise is interfering 
+with the communication bus during each of the electrical transitions. One may also note how the motor noise on the frame has a DC (the “on” and “off” voltage at +/-2.5 V) 
+and an AC (the transition between the two DC states) component, but only the AC component is coupling via the stray capacitance.
+
+How To Minimize Noise On Communication Lines
```

Comment:
I still think that Frame Grounding should be a subsection of this. What do y'all think?

Reply:
It can be. The main reason I had it as it's own section is that the "Frame Grounding" section is so big that it is half of the article.

Reply:
That makes sense. I just think that if we ever have a new section describing another method of noise minimization, we would want it to under the general branch of "How to Minimize..." as well.

Reply:
@iq-fred can we get a tie breaker?

Reply:
@jteoli4 @iq-fred I agree that it makes sense as a subsection of the how to minimize noise section. I'd say subsection if it is not too annoying